src/history/hs_rec.c

```diff
@@ -491,8 +455,6 @@ __wt_hs_insert_updates(WT_SESSION_IMPL *session, WT_PAGE *page, WT_MULTI *multi)
             /* Track the first update that is globally visible. */
             if (first_globally_visible_upd == NULL && __wt_txn_upd_visible_all(session, upd))
                 first_globally_visible_upd = upd;
-            else if (first_globally_visible_upd != NULL)
```

Comment:
Do we now insert updates behind the first globally visible upd to the history store?

Reply:
See https://github.com/wiredtiger/wiredtiger/pull/6091#issuecomment-715337437.

We already have logic which skips inserting globally visible updates.

Reply:
That only skips them if they are not modifies. If the first globally visible update is a modify and we have a series of modifies behind it. I think the change will insert them to the history store.

Reply:
We're going to leave it as is for now. To be fixed in WT-6837